RS-Project-6/LcmAndHcfCalculate/LcmAndHcfCalculateTest.java

```diff
@@ -0,0 +1,67 @@
+import static org.junit.Assert.*;
+
+import org.junit.Test;
+
+public class LcmAndHcfCalculatorTest {
+    LcmAndHcfCalculator calculator = new LcmAndHcfCalculator();

```

Comment:
As per standards, the variable name should likely to be similar to class name and instead of defining the class variable here, instantiate those in particular test method.